networking_aci/plugins/ml2/drivers/mech_aci/rpc_api.py

```diff
@@ -96,13 +94,13 @@ def get_networks(self, rpc_context, limit=None, marker=None):
 
     @log_helpers.log_method_call
     def get_network_ids(self, rpc_context):
-        return self.db.get_network_ids(self.context)
+        return self.db.get_network_ids(rpc_context)
 
     @log_helpers.log_method_call
     def get_networks_count(self, rpc_context):
-        return self.db.get_networks_count(self.context)
+        return self.db.get_networks_count(rpc_context)
 
-    def _get_network(self, network):
+    def _get_network(self, context, network):
```

Comment:
would call it `rpc_context` to be consistent with the other functions and the callers here
same below for `ACIRpcClientAPI` and  `AgentRpcClientAPI` where it was called `rpc_context` before

Reply:
In contrast to the other methods `_get_network()` is not called via RPC, but only used by other methods. It doesn't necessarily need an "rpc_context" (though this is (at the moment?) the same thing as a normal context). This method is also used in one other place in driver.py to acquire sync data, which isn't an rpc call. Therefore I'd leave it the way it is.

From a code layout perspective this method also shouldn't be here, it should probably be part of our own DPlugin (see common.py), but in this PR I wanted to get the context mess sorted out and not do too much of reorganization.